src/service.rs

```diff
@@ -462,25 +485,53 @@ impl CollectionManager {
         })
     }
 
+    /// Create a new encrypted [Collection] and return a handle to it
+    ///
+    /// # Arguments:
+    /// * `collection_type` - the type of [Item]s stored in the collection
+    /// * `meta` - the [ItemMetadata] for the collection  
+    /// * `content` - the collection's content as a byte array. This is unrelated to the [Item]s in the collection. 
     pub fn create<T: MsgPackSerilization>(&self, collection_type: &str, meta: &T, content: &[u8]) -> Result<Collection> {
         let meta = meta.to_msgpack()?;
         self.create_raw(collection_type, &meta, content)
     }
 
+    /// Create a new encrypted [Collection] and return a handle to it. This receives the metadata as a byte array.
```

Comment:
I think it's better to split it to two lines. Potentially with an empty line in between (like git) to better emphasise the first sentence as the main one.

```
/// Create a new encrypted [Collection] and return a handle to it using raw metadata
///
/// Unlike [create] metadata is expected to be a valid msgpack-encoded metadata object.
///
/// # Arguments:
/// ...